config/_default/languages.toml

```diff
@@ -15,4 +15,7 @@ languageName = "Русский"
 weight =  5
 [tr]
 languageName = "Türkçe"
-weight =  6
\ No newline at end of file
+weight =  6
+[de]
+languageName = "German"
```

Comment:
This should be "Deutsch" not "German". The idea is to make it more obvious to those language speakers that its their language.

Reply:
Good catch! Didn't realize the word was not German in German.

Updated it!